oauth2client/contrib/django_util/__init__.py

```diff
@@ -36,9 +46,13 @@
 
     INSTALLED_APPS = (
         # other apps
+        "django.contrib.sessions.middleware"
         "oauth2client.contrib.django_util"
     )
 
+This helper also requires the Django Session Middleware, so
+`django.contrib.sessions.middleware` should be in INSTALLED_APPS as well.
```

Comment:
two backticks so that Sphinx formats it as code (one backtick is for italic, apparently).